config/tomcat.yml

```
@@ -40,5 +40,5 @@ redis_store:
   timeout: 2000
   connection_pool_size: 2
 geode_store:
-  version: 1.11.+
+  version: 1.11.0
```

```suggestion
  # The version of Geode Store must be less than or equal to the Geode server version to ensure compatibility.
  #  version is pinned to 1.11.0 to match the oldest supported version of Tanzu Gemfire for VMs
  version: 1.11.0
```
Sorry, I meant this kind of comment.

---

Yeah, I was thinking about that this morning and realized that's probably what you meant 😅

---

Just updated it, but let me run the comment by TGF4VMs PMs before merging.

---

Done! Let us know what you think.